products/laravel.md

```diff
@@ -12,41 +12,48 @@ sortReleasesBy: 'releaseCycle'
 auto:
   git: https://github.com/laravel/framework.git
 releases:
-  - releaseCycle: "9"
+-   releaseCycle: "9"
     release: 2022-02-08
     support: 2023-08-08
     eol: 2024-02-08
     latest: 9.12.2
```

Comment:
This should be 9.13.0

https://github.com/endoflife-date/release-data/blob/58c73317232e7012d8abac04478dd261b705e051/releases/git/laravel.json#L780

Reply:
This commit isn't yet merged here ([Last PR](https://github.com/endoflife-date/endoflife.date/pull/1197) 17 hours ago updated to `cbfc3ed13fd593cb5c8cc22393d715b421d65279`). So this should get updated once dependabot files the next PR.

Reply:
Definitely a timezone issue. It looks like Ruby/Rugged doesn't preserve timezone information from dates read from Git. This is what I see in my local repo:

```
tag mariadb-10.7.3
Tagger: Daniel Bartholomew <db@dbart.us>
Date:   Sat Feb 12 13:38:44 2022 -0500
```

vs what I get in rugged:

`10.7.3: TZ=IST 2022-02-11 01:46:50 +0530`

Checking further.